etc/links.profile

```diff
@@ -0,0 +1,59 @@
+# Firejail profile for links
+# Description: Text WWW browser
+# This file is overwritten after every install/update
+#
+# Persistent local customizations
+include links.local
+# Persistent global definitions
+# include globals.local
+
+noblacklist ${HOME}/.links
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.links
+whitelist ${HOME}/.links
+whitelist ${DOWNLOADS}
+
+caps.drop all
+ipc-namespace
+# uncomment machine-id if you want to allow access only to user-configured
+# associated music player
+machine-id
+netfilter
+# uncomment no3d if you want to allow access only to user-configured
+# associated video player
+no3d
+nodvd
+nogroups
+nonewprivs
+noroot
+# uncomment nosound if you want to allow access only to user-configured
+# associated music player
+nosound
+notv
+nou2f
+# uncomment novideo if you want to allow access only to user-configured
+# associated video player
+novideo
+protocol unix,inet,inet6
+seccomp
+shell none
+tracelog
+# uncomment to allow user-configured associated programs to use X11
+x11 none
```

Comment:
For me (Fedora 29):
```
firejail --profile=links.profile links 
Reading profile links.profile
…
ERROR: --x11=none specified, but abstract X11 socket still accessible.
Additional setup required. To block abstract X11 socket you can either:
 * use network namespace in firejail (--net=none, --net=...)
 * add "-nolisten local" to xserver options
   (eg. to your display manager config, or /etc/X11/xinit/xserverrc)
zsh: exit 1     firejail --profile=links.profile links
```
remove

Reply:
same here (me talking about profile testing..) on arch. ok.. so it means we're going back to `blacklist /tmp/.X11-unix`?